snappy/snap_yaml.go

```diff
@@ -66,7 +66,7 @@ type AppYaml struct {
 	Description string          `yaml:"description,omitempty" json:"description,omitempty"`
 	Stop        string          `yaml:"stop,omitempty"`
 	PostStop    string          `yaml:"poststop,omitempty"`
-	StopTimeout timeout.Timeout `yaml:"stop-timeout,omitempty"`
+	StopTimeout timeout.Timeout `yaml:"stop_timeout,omitempty"`
```

Comment:
actually, it seems this is json/yaml, so it would need double defs, missing tests?